include/fmt/base.h

```diff
@@ -8,13 +8,15 @@
 #ifndef FMT_BASE_H_
 #define FMT_BASE_H_
 
-#include <limits.h>  // CHAR_BIT
-#include <stdio.h>   // FILE
-#include <string.h>  // strlen
+#ifndef FMT_IMPORT_STD
+#  include <limits.h>  // CHAR_BIT
+#  include <stdio.h>   // FILE
+#  include <string.h>  // strlen
```

Comment:
These includes should be moved out of the ifndef blocks because they are not part of the std module.